modules/hibernating-cluster-hibernate.adoc

```diff
@@ -0,0 +1,97 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/hibernating-cluster.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="hibernating-cluster-hibernate_{context}"]
+= Hibernating a cluster
+
+You can hibernate a cluster for up to 90 days. The cluster can recover if certificates expire while the cluster was in hibernation.
+
+.Prerequisites
+
+* The cluster has been running for at least 24 hours to allow the first certificate rotation to complete.
++
+[IMPORTANT]
+====
+If you must hibernate your cluster before the 24 hour certificate rotation, use the following procedure instead: link:https://www.redhat.com/en/blog/enabling-openshift-4-clusters-to-stop-and-resume-cluster-vms[Enabling OpenShift 4 Clusters to Stop and Resume Cluster VMs].
```

Comment:
Some of the content mentioned in this paragraph is repeated [there](https://github.com/openshift/openshift-docs/pull/86873/files#diff-2c11a25f58a35f2890cd67f5b9b7bfe9e195521bdff39d7b2e26a5d91d9b9d4dR11-R15), so we can consider keeping only the version here

Reply:
I'd probably prefer to keep it both places. There is a chance that someone might read one section or another. I think it's an important enough thing to know that it shouldn't hurt to mention twice.

Reply:
OK.